x-pack/plugins/security/public/management/api_keys/api_keys_grid/api_key_flyout.tsx

```diff
@@ -720,6 +914,13 @@ export const ApiKeyFlyout: FunctionComponent<ApiKeyFlyoutProps> = ({
   );
 };
 
+const TypeLabel = () => (
+  <FormattedMessage
+    id="xpack.security.accountManagement.apiKeyFlyout.typeLabel"
+    defaultMessage="Type"
+  />
+);
```

Comment:
Nit: could implement similar const functions for the other reused labels/text/links: name, invalidJsonError, DocLink

Reply:
Consolidated translation strings

(couldn't really do that for DocLink as those were all unique)